src/Http/Middleware/HttpsEnforcerMiddleware.php

```diff
@@ -94,4 +106,24 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             'Requests to this URL must be made with HTTPS.'
         );
     }
+
+    /**
+     * Adds Strict-Transport-Security header to response.
+     *
+     * @param \Psr\Http\Message\ResponseInterface $response Response
+     * @return \Psr\Http\Message\ResponseInterface
+     */
+    protected function addHsts(ResponseInterface $response): ResponseInterface
+    {
+        $config = $this->config['hsts'];
```

Comment:
Should we raise an error if the config data isn't an array? Someone will eventually make the value a boolean or a string.

Reply:
Sure. It would have thrown a php error, but not it's an UnexpectedValueException with message.